pkg/cmd/bootstrap.go

```diff
@@ -225,10 +225,29 @@ func checkBootstrapDependencies(io *BootstrapParameters, client *utility.Client,
 
 	spinner.Start("Checking if Sealed Secrets is installed with the default configuration", false)
 	if err := client.CheckIfSealedSecretsExists(defaultSealedSecretsServiceName); err != nil {
-		warnIfNotFound(spinner, "Please install Sealed Secrets Operator from OperatorHub", err)
-		if !apierrors.IsNotFound(err) {
-			return fmt.Errorf("failed to check for Sealed Secrets Operator: %w", err)
+
+		// in case Sealed Secret namespace/service name are provided, try them too
+		if (len(io.BootstrapOptions.SealedSecretsService.Namespace) > 0 && io.BootstrapOptions.SealedSecretsService.Namespace != defaultSealedSecretsServiceName.Namespace) ||
```

Comment:
I don't like to use string length comparisons to check if a string is empty, it compiles to the same as testing for `""` and it's a lot more obvious what is actually being compared.

Reply:
yes, will change to `!= ""`